cyclonedx/model/__init__.py

```diff
@@ -492,11 +492,11 @@ class ExternalReference:
         See the CycloneDX Schema definition: https://cyclonedx.org/docs/1.3/#type_externalReference
     """
 
-    def __init__(self, *, type_: ExternalReferenceType, url: XsUri, comment: Optional[str] = None,
+    def __init__(self, *, type: ExternalReferenceType, url: XsUri, comment: Optional[str] = None,
```

Comment:
At this point, it would be better to have the variable (not the property) called `type_` 
because here in the scope, it would shadow pythons function `type()`.
As long as we don't call `type()` in this method scope, it is safe.

Reply:
Thanks.